deploy/helm/kubecf/assets/operations/instance_groups/database.yaml

```diff
@@ -105,8 +118,23 @@
 - type: replace
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/roles/name=cloud_controller/name
   value: &external_cc_database_username {{ .Values.features.external_database.databases.cc.username | quote }}
+{{ if .Values.features.external_database.ca_cert -}}
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/ca_cert?
+  value: {{ toYaml .Values.features.external_database.ca_cert | indent 2 }}
+{{- else }}
 - type: remove
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/ca_cert?
+{{- end}}
+{{ if .Values.features.external_database.require_ssl -}}
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/ssl_verify_hostname?
+  value: true
+{{- else }}
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/ssl_verify_hostname?
+  value: true
```

Comment:
This seemed wrong, but the suggestion above should fix it.

Reply:
Yupp looks like some bad copy-paste action that wasnt cleaned up in the PR commit